rust/plugin-lib/src/lib.rs

```diff
@@ -74,6 +74,10 @@ pub trait Cache {
     /// [`DataSource`]: trait.DataSource.html
     fn get_line<DS: DataSource>(&mut self, source: &DS, line_num: usize)
         -> Result<&str, Error>;
+    
+    /// Returns the whole document in the current View
```

Comment:
This documentation is for the `Cache` trait, which doesn't have any concept of a view. Instead here I would say something like,

"Returns the entire contents of the remote document, fetching as needed."